MetaMask/metamask-mobile
# test: build new Ramps quotes

## **Description**

This tests that the user can change options in the Build Quote screen and build new quotes. The new quotes should show the updated currency/token/fiat/etc. 

## **Related issues**

https://consensyssoftware.atlassian.net/browse/RAMPS-1933

## **Manual testing steps**

All Ramps tests are passing:
```
 PASS  e2e/specs/ramps/onramp.spec.js (185.993 s)
  smokeAssets Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (17561 ms)
    ✓ should select a new region and check the quotes (44459 ms)
    ✓ should select a new token and check the quotes (35913 ms)
    ✓ should select a new currency and check the quotes (33345 ms)
    ✓ should select a new payment method and check the quotes (33233 ms)

Test Suites: 1 passed, 1 total
Tests:       5 passed, 5 total
```

## **Screenshots/Recordings**

Uploading Untitled.mov…


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
